scripts/superres_pipeline_custom_data.py

```diff
@@ -0,0 +1,121 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
@tge25 Do we want to merge this? I think it's a nice example. Maybe we can include an ncdump of the .zarr data so users know what format to comply with.